.github/workflows/build-mac.yml

```diff
@@ -0,0 +1,114 @@
+name: Build installable mac distributions
+#on: workflow_dispatch
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-distribution:
+#    environment: release
+    strategy:
+      matrix:
+        os: [ macos-11, macos-12 ]
```

Comment:
Why do you need two OS builds? Arelle currently distributes a single mac build that works on macOS 10.15, 11, and 12.

Reply:
The behavior is different between the 2 operating systems I have found (UI uses different look and feel) I am not sure its strictly necessary, but seems safer since you will have users for multiple OS. I will defer to you though, I am completely fine just building latest

Reply:
Good to know. Let's stick to just macos-12 for now. If there are issues we can add more, but I'd rather not support/have to test additional builds if we don't strictly need to.